indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -36,7 +36,9 @@
                 var data_href = $(this).data('href');
                 href = data_href ? data_href : href;
             }
-            ajaxDialog($.extend({}, options, {
+            ajaxDialog($.extend({
+                title: $(this).data('title')
```

Comment:
I remember I proposed this some time ago but there was some problem. Do you remember?